### DISCUSSION TITLE
My Sponsors profile got deactivated

### DISCUSSION THREAD

**Discussion by krasa** [community/community#52154] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Don't know what happened.

My dashboard says:
```
Your profile isn't live yet. We need more information to continue.
Submit application to GitHub staff for approval
```
![image](https://user-images.githubusercontent.com/1160875/230351272-a3d38850-a4d4-4391-b4f9-1ad873665fbb.png)

`Request approval` says: `Your profile cannot be submitted for approval at this time.  `
![image](https://user-images.githubusercontent.com/1160875/230350788-9459d062-286d-442c-aba8-4784f3f545f8.png)

**Comment by BREN0sx** [comment, marked as answer]
This error can have several causes, including but not limited to: incomplete or invalid profile information, lack of recent activity on GitHub, use of an unverified email address or one associated with another account, attempting to become a sponsor on an account with less than 18 years old, violation of GitHub's terms of service or sponsorship program, among other factors. The best way to resolve the issue is to contact GitHub support for specific assistance.

**Reply by krasa** [reply]
Thanks. It was approved again without me doing anything :-) 
I only had some strange billing information related to plans (which I never used), not related to sponsors.

**Reply by huynhvienan** [reply]
https://github.com/orgs/community/discussions/52154#discussioncomment-5545552

**Comment by huynhvienan** [comment]
Yeah
